.github/workflows/docker-layers.yml

```diff
@@ -0,0 +1,323 @@
+name: Docker layers
```

Comment:
I'm not sure I follow what this is supposed to do? I think `publish-build-layer-images.yml` is supposed to do this (and looks cleaner and is better named)? So perhaps this needs deleting?